src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Truck(), new Excavator(), new Bulldozer()};
```

Comment:
Consider using abstract references instead of specific ones. This would allow the code to be more flexible and take advantage of polymorphism. For example: 'Machine truck = new Truck();' instead of directly instantiating a 'Truck'.